forced_photometry/multiband_photometry.md

```diff
@@ -854,6 +839,26 @@ rad_in_arcmin = radius.value  #units attached to this are confusing nway down th
 nway_write_header('data/multiband_phot.fits', 'OPT', float((2*rad_in_arcmin/60)**2) )
 ```
 
+```{raw-cell}
+#fix the first line of the nway code which calls python
+path = "/home/jovyan/.local/bin/nway.py"
+
+# Read the file, replace first line, and write it back
+with open(path, 'r') as f:
+    lines = f.readlines()
+
+if not lines[0].startswith('#!'):
+    lines.insert(0, '#!/usr/bin/env python3\n')
+else:
+    lines[0] = '#!/usr/bin/env python3\n'
+
+with open(path, 'w') as f:
+    f.writelines(lines)
+
+# Make sure it's still executable
+!chmod +x /home/jovyan/.local/bin/nway.py
+```
+
```

Comment:
Could we just call it explicitly with python rather than hacking around in the file itself?